packages/playwright-test/src/runner.ts

```diff
@@ -237,148 +227,194 @@ export class Runner {
       files.forEach(file => allTestFiles.add(file));
 
     const config = this._loader.fullConfig();
-    const internalGlobalTeardowns: (() => Promise<void>)[] = [];
-    if (!list) {
-      for (const internalGlobalSetup of this._internalGlobalSetups)
-        internalGlobalTeardowns.push(await internalGlobalSetup());
+
+    const fatalErrors: TestError[] = [];
+
+    // 1. Add all tests.
+    const preprocessRoot = new Suite('');
+    for (const file of allTestFiles) {
+      const fileSuite = await this._loader.loadTestFile(file);
```

Comment:
It is somewhat scary that we continue loading files after an error, because this could trigger cascading errors.

For example:
```js
// When "base" module did not load because of syntax error or something,
// "Base" will be undefined and extending it will fail with a cascading error.

const { Base } = require('./base');
const Derived extends Base {}
```

Also, if I have a syntax error in the fixtures file, every single test file will get that error reported, won't it?